spec/classes/init_spec.rb

```diff
@@ -484,8 +484,10 @@
         end
         let(:params) {{ :login_pam_access => 'sufficient' }}
 
-        if (v[:osfamily] == 'RedHat') or (v[:osfamily] == 'Suse' and ['10', '11', '12'].include?(v[:release]))
+        if (v[:osfamily] == 'RedHat') or (v[:osfamily] == 'Suse' and ['10', '11', '12'].include?(v[:release])) or (v[:osfamily] == 'Debian' and ['7', '8'].include?(v[:release])) or (v[:osfamily] == 'Debian' and v[:lsbdistid] == 'Ubuntu' and ['12.04', '14.04', '16.04'].include?(v[:release]))
```

Comment:
I didn't did the math, maybe the number of else cases is smaller than this list ? Maybe turning around the else case is easier to read ?